vllm/v1/metrics/loggers.py

```diff
@@ -229,6 +262,16 @@ def log(self):
         if not self.connector_prefix_caching_metrics.empty:
             log_parts.append("External prefix cache hit rate: %.1f%%")
             log_args.append(self.connector_prefix_caching_metrics.hit_rate * 100)
+
+        if self.has_perf_stats and self._enable_perf_stats():
```

Comment:
Can't we just make this conditional on `last_avg_tflops_per_gpu` and `last_avg_gbps_per_gpu` being non-zero?

Reply:
Done, but keeping _enable_perf_stats() for reasons mentioned in https://github.com/vllm-project/vllm/pull/28859#discussion_r2557955671